docs/components/input-controller/_index.md

```diff
@@ -44,281 +44,475 @@ Once you've configured your input controller according to model type, you can wr
-    logger.Info(resp)
+This method returns the current state of the controller as a map of [Event Objects](#event-object), representing the most recent event that has occured on each available [Control](#control-field).
 
-    // Log any errors that occur.
-    if err != nil {
-        logger.Fatal(err)
-    }
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
 
-    // Run the handleController function.
-    err := HandleController(myController)
+- `extra` [(Optional[Dict[str, Any]])](https://docs.python.org/library/typing.html#typing.Optional): Extra options to pass to the underlying RPC call.
+- `timeout` [(Optional[float])](https://docs.python.org/library/typing.html#typing.Optional): An option to set how long to wait (in seconds) before calling a time-out and closing the underlying RPC call.
 
-    // Delay closing your connection to your robot.
-    err = myRobotWithController.Start(ctx)
-    defer myRobotWithController.Close(ctx)
+**Returns:**
 
-    // Watch for errors.
-    if err != nil {
-        return err
-    }
+- `events` [(Dict[Control, Event])](https://docs.python.org/3/library/typing.html#typing.Dict): A dictionary mapping the most recent Event for each Control.
 
-    // Wait to exit mainWithArgs() until Context is Done.
-    <-ctx.Done()
+For more information, see the [Python SDK Docs](https://python.viam.dev/_modules/viam/components/input/input.html#Controller.get_events).
 
-    return nil
+```python {class="line-numbers linkable-line-numbers"}
+# Get the most recent Event for each Control.
+recent_events = await myController.get_events()
 
-}
+# Print out the most recent Event for each Control.
+print(f'Recent Events:\n{recent_events}')
 ```
 
 {{% /tab %}}
-{{< /tabs >}}
+{{% tab name="Go" %}}
 
-{{% alert title="Note" color="note" %}}
-The `Controller` interface is defined in [the Viam RDK](https://github.com/viamrobotics/rdk/blob/main/components/input/input.go).
+**Parameters:**
 
-{{% /alert %}}
+- `Context` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+- `extra` [(map[string]interface{})](https://pkg.go.dev/google.golang.org/protobuf/types/known/structpb): Extra options to pass to the underlying RPC call.
 
-For more examples see [Usage Examples](#usage-examples).
+**Returns:**
 
-## Types
+- `events` [(map[Control]Event)](https://pkg.go.dev/go.viam.com/rdk/components/input#Control): A map mapping the most recent Event for each Control.
+- `error` [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
 
-### Event Object
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/input#Controller).
 
-Each `Event` object represents a singular event from the input device, and has four fields:
+```go {class="line-numbers linkable-line-numbers"}
+// Get the most recent Event for each Control.
+recent_events, err := myController.Events(context.Background(), nil)
```

Comment:
```suggestion
recent_events, err := my_controller.Events(context.Background(), nil)
```

Reply:
This is just go convention vs python convention here!